src/UserHome/UserHome.js

```diff
@@ -2,16 +2,14 @@ import React, { Component } from "react";
 import MovieContainer from '../MovieContainer/MovieContainer';
 import { Link } from "react-router-dom";
 
-const UserHome = ({ appState, getUsersRatings}) => {
-	getUsersRatings(appState.userID);
```

Comment:
this was setting an infinite loop of api calls being made. it should either be made in a componentDidMount lifecycle, or in some other conditional. added it to only be called once at the resolved promise of a successful login. above lin 38 on LoginPage.js